<?xml version="1.0"?>
<launch>
    <arg name="room" default="cave" /> 

    <!-- Map Server -->
    <include file="$(find tuw_voronoi_graph)/launch/map_server.launch">
        <arg name="room" value="$(arg room)"/>
    </include>

    <!-- Frontier Generator -->
    <node name="frontiers_generator_node" pkg="roadmap_builder" type="frontiers_generator.py" output="screen"/>

    <!-- Voronoi Graph Generator -->
    <include file="$(find tuw_voronoi_graph)/launch/graph_generator.launch">
        <arg name="room" value="$(arg room)"/>
    </include>

    <!-- Roadmap Builder -->
    <node name="roadmap_builder_node" pkg="roadmap_builder" type="roadmap_builder_node.py" output="screen"/>

</launch>